### PR TITLE
Announce SciPy 1.7.2

### PR DESCRIPTION
**Pull request by tylerjereddy**
Announce SciPy `1.7.2` release.

Let me know if I've done something wrong so I can learn the new process.

### CODE REVIEW

**Comment by tylerjereddy** [comment]
whoa, I can click the link to check for myself, pretty cool

File: content/en/news.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ title: News
 sidebar: false
 ---
 
+### SciPy 1.7.2 released
+
+_November 5, 2021_ -- SciPy 1.7.2 is a bug-fix release with no new features compared to 1.7.1.
```
Comment (rgommers) [inline content/en/news.md:8]: Can you add that this is the first release to add Python 3.10 support? That's an important feature I'd say!

Comment (tylerjereddy) [reply content/en/news.md]: ok, I tried to revise the news item accordingly

**Review by jarrodmillman — APPROVED** [review]

**Comment by tupui** [comment]
Merged so the official site shows the release.

**Pull request merged**